docs/source/UIDocumentation.md

```diff
@@ -1,160 +1,235 @@
+* ```mouseClick``` - invoked on mouse click; toggles model categories
+* ```componentDidMount``` - mounts the model search code
+* ```render``` - renders the modelZoo
+
+***
+
+#### ```netLayout_vertical.js``` and ```netLayout.js```
 Both of these files contain code that determines positioning and layout of net. It is invoked by ```content.js```
-### ```setParams.js```
-Contains the following methods:
-* ```changeProps```
-    * prop
-    * value
-* ```changeParams```
-    * para
-    * value
-* ```close```
-* ```trainOnly```
-    * e
-    
-### ```setParams.js``` 
-allows users to change layer parameters and reads paramters through ```data.js```
-### ```tooltip.js``` and ```tooltipData.js```
+
+#### ```netLayout_vertical.js```
+Defines the vertical alignment of network elements. ```netLayout_vertical.js``` uses a function, called ```allocatePosition``` to give the layers their position on the canvas. It finds the closest position available to the preferred position, and checks if a node is already on that position on the X axis (if one is, a position left or right is assigned). The file also uses a custom Topolical Sort to give the nodes their position (it is based on the inputs and outputs of nodes). Later it is checked if any nodes overlap and in case they do, they are moved.
+
+#### ```netLayout.js```
+```netLayout.js``` is used to check the Y attribute of nodes. The function ```allocatePosition``` finds the closest available position to the preffered one and allocates the module. Later the nodes are checked with a DFS algorithm (depth-first search algorithm).
+
+***
+
+#### ```pane.js```
+```pane.js``` handles the layer panel and contains these methods:
+* ```toggleClass``` - toggles classes for the dropdown on the sidebar for layer selection
+* ```componentDidMount``` - handles element with layers: search for layers, shown elements, etc.
+* ```render``` - renders layer panel
+
+```pane.js``` invokes ```paneElement.js``` to render out each element.
+
+***
+
+#### ```paneElement.js```
+```paneElement.js``` renders out each element of the pane, it is invoked by ```pane.js```. It includes:
+* ```drag``` - allows dragging of layers
+* ```render``` - renders specific elements
+
+```pane.js``` renders out all of the layers for selection by the user.
+***
+#### ```panZoom.js```
+```panZoom.js``` includes the functions that zoom the canvas in and out, based on various invocations.
+
+***
+
+#### ```setParams.js```
+Contains the following methods for the layer edition sidebar:
+* ```changeProps``` - changes layer properties
+* ```changeParams``` - changes layer parameters
+* ```close``` - closes sidebar
+* ```trainOnly``` - turns model into a train-only
+* ```handleKeyPress``` - handles layer delete after the 'delete' key is pressed
+* ```componentDidMount``` - invokes ```handleKeyPress```
+* ```componentWillUnmount``` - invokes ```handleKeyPress```
+* ```render``` - renders layer options sidebar
```

Comment:
Please describe only methods defined by us, no need to describe standard react methods